.doc_gen/metadata/cross_metadata.yaml

```diff
@@ -289,3 +289,50 @@ cross_RekognitionVideoDetection:
             sqs:
   services:
     rekognition:
+cross_DetectFaces:
+  title: Detect faces in an image using an &AWS; SDK
+  title_abbrev: Detect faces in an image
+  synopsis_list:
+    - Save an image in an &S3long; (&S3;) bucket.
+    - Displays facial details, such as age range, gender, and emotion (smiling, etc.).
+  languages:
+    Rust:
+      versions:
+        - sdk_version: 1
+          block_content: cross_detect_faces_rust_block.xml
+  services:
+    s3: {}
+cross_DetectLabels:
+  title: Save EXIF and other image information using an &AWS; SDK
+  title_abbrev: Save EXIF and other image information
```

Comment:
I had to look up EXIF, so maybe not good in the title. Maybe something more like:
`Detect image information and save it to a &DDB; database`